jdaviz/configs/imviz/tests/test_linking.py

```diff
@@ -219,52 +221,71 @@ def test_wcslink_rotated(self):
+                        [[-1.590838, -0.423662],
+                         [-1.826862, 9.896219],
+                         [8.590838, 9.423662],
+                         [8.826862, -0.896219]], rtol=1e-5)
         assert_allclose(gwcs_zoom_limits,
-                        ((3.245117, 10.549265), (10.688389, 4.95208),
-                         (6.100057, -2.357782), (-1.343215, 3.239403)), rtol=1e-5)
-        assert_allclose(no_wcs_zoom_limits, fits_wcs_zoom_limits)
+                        [[3.186753, 11.337468],
+                         [11.895862, 5.072343],
+                         [6.158421, -3.145985],
+                         [-2.550688, 3.119141]], rtol=1e-5)
 
         # Also check the coordinates display: Last loaded is on top.
-        # Cycle order: no_wcs, FITS WCS, GWCS
-
+        # Cycle order: FITS WCS, GWCS
         label_mouseover = self.imviz.app.session.application._tools['g-coords-info']
-        label_mouseover._viewer_mouse_event(self.viewer,
-                                            {'event': 'mousemove', 'domain': {'x': 0, 'y': 0}})
-        assert label_mouseover.as_text() == ('Pixel x=00.0 y=00.0 Value +1.00000e+00', '', '')
-        assert not label_mouseover.row1_unreliable
-        assert not label_mouseover.row2_unreliable
-        assert not label_mouseover.row3_unreliable
 
+        x, y = _transform_refdata_pixel_coords(
+            self.imviz.app.data_collection[-1], self.imviz.app.data_collection[1],
+            in_x=0, in_y=0
+        )
+        label_mouseover._viewer_mouse_event(self.viewer,
+                                            {'event': 'mousemove', 'domain': {'x': x, 'y': y}})
+        assert label_mouseover.as_text()[0] in ('Pixel x=00.0 y=00.0 Value +1.00000e+00',
+                                                'Pixel x=-0.0 y=00.0 Value +1.00000e+00',
+                                                'Pixel x=-0.0 y=-0.0 Value +1.00000e+00',
+                                                'Pixel x=00.0 y=-0.0 Value +1.00000e+00')
+        # assert not label_mouseover.row1_unreliable
+        # assert not label_mouseover.row2_unreliable
+        # assert not label_mouseover.row3_unreliable
+
+        x, y = _transform_refdata_pixel_coords(
+            self.imviz.app.data_collection[-1], self.imviz.app.data_collection[0],
+            in_x=0, in_y=0
+        )
         self.viewer.on_mouse_or_key_event({'event': 'keydown', 'key': 'b',
-                                           'domain': {'x': 0, 'y': 0}})
-        assert label_mouseover.as_text() == ('Pixel x=00.0 y=00.0 Value +1.00000e+00 electron / s',
-                                             'World 00h14m19.6141s -30d23m31.4091s (ICRS)',
-                                             '3.5817255823 -30.3920580740 (deg)')
+                                           'domain': {'x': x, 'y': y}})
+        assert label_mouseover.as_text() == ('Pixel x=09.8 y=02.8',

```

Comment:
I cannot wrap my head around this change in the tested behavior. What is going on here?

Reply:
Fixed this too.